src/TwigComponent/src/ComponentFactory.php

```diff
@@ -189,6 +208,35 @@ private function postMount(object $component, array $data): array
         return $data;
     }
 
+    private function isAnonymousComponent(string $name): bool
+    {
+        return null !== $this->findAnonymousComponentTemplate($name);
+    }
+
+    public function findAnonymousComponentTemplate(string $name): ?string
+    {
+        $loader = $this->environment->getLoader();
+        $componentPath = rtrim(str_replace(':', '/', $name));
+
+        if ($loader->exists($componentPath)) {
+            return $componentPath;
+        }
+
+        if ($loader->exists($componentPath.'.html.twig')) {
+            return $componentPath.'.html.twig';
+        }
+
+        if ($loader->exists('components/'.$componentPath)) {
+            return 'components/'.$componentPath;
+        }
+
+        if ($loader->exists('/components/'.$componentPath.'.html.twig')) {
+            return '/components/'.$componentPath.'.html.twig';
```

Comment:
Are the starting `/` there on purpose? I'm not familiar with Twig template paths starting with `/`